src/ctap/storage.rs

```diff
@@ -30,16 +32,16 @@ use core::convert::TryInto;
 use crypto::rng256::Rng256;
 
 #[cfg(any(test, feature = "ram_storage"))]
-type Storage = embedded_flash::BufferStorage;
+type Storage = persistent_store::BufferStorage;
 #[cfg(not(any(test, feature = "ram_storage")))]
-type Storage = embedded_flash::SyscallStorage;
+type Storage = crate::embedded_flash::SyscallStorage;
 
 // Those constants may be modified before compilation to tune the behavior of the key.
 //
-// The number of pages should be at least 2 and at most what the flash can hold. There should be no
-// reason to put a small number here, except that the latency of flash operations depends on the
-// number of pages. This will improve in the future. Currently, using 20 pages gives 65ms per
-// operation. The rule of thumb is 3.5ms per additional page.
+// The number of pages should be at least 3 and at most what the flash can hold. There should be no
+// reason to put a small number here, except that the latency of flash operations is linear in the
+// number of pages. This may improve in the future. Currently, using 20 pages gives between 20ms and
+// 240ms per operation. The rule of thumb is between 1ms and 12ms per additional page.
```

Comment:
This is quite a big latency. Given that we have some uncached implementations of counting and such, this sounds like commands can take a second in the store alone. Did you notice any lags in practise?

I assume most read-only operations are actually closer to 20ms?

Reply:
My comment wasn't really actionable (at least not in this PR), just curious. A long latency have some implications on sending keepalive packets, that's why I'm asking.